src/frontend/apps/impress/src/features/docs/doc-tree/hooks/useTreeItemKeyboardActivate.tsx

```diff
@@ -0,0 +1,8 @@
+import { useKeyboardActivation } from './useKeyboardActivation';
+
+export const useTreeItemKeyboardActivate = (
```

Comment:
You created the component `useKeyboardActivation`, is this component really necessary anymore ? If yes they can be merged in a single file I think.